templates/4a_compensate_image_data.ipynb

```diff
@@ -115,6 +115,9 @@
     "rosetta_testing_dir = 'C:\\\\Users\\\\Customer.ION\\\\Documents\\\\rosetta_testing'\n",
```

Comment:
<p>This comment should go in the cell where run_names is defined, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>